src/maintainer/adding_pkgs.rst

```diff
@@ -3,71 +3,94 @@
+      #. Fork and clone the `staged-recipes <https://github.com/conda-forge/staged-recipes>`_ repository from GitHub.
+      #. Checkout a new branch from the staged-recipes ``main`` branch.
+      #. Through CLI, cd inside the 'staged-recipes/recipes' directory.
+      #. Call grayskull and pass the pypi repository, followed by the name of the package you want to contribute to conda-forge. (i.e ``grayskull pypi <name-of-package>`` **OR** Use grayskull ``pypi <name-of-package> --strict-conda-forge`` to remove some selectors which are not necessary for conda-forge and adapt recipes to fit better in the conda-forge ecosystem.
+        Grayskull will create a folder with the same name as the package in the ‘recipes’ folder of the ‘staged-recipes’ directory. This folder will contain the meta.yaml file and also the license file if the package includes a license in the PyPI distribution.
+      #. Go through the generated ``meta.yaml`` file and remove all irrelevant comments in the ``meta.yaml``  file.. For simpler packages, the generated recipes are nearly perfect, but for some packages you might need to make certain tweaks.
+      #. Commit and push the changes. git add recipe/abc/meta.yaml git commit -m "add a commit message" git push
+      #. Create a PR.
+
+      Know more about `Grayskull <https://github.com/conda-incubator/grayskull>`_
+
+   .. tab:: Example recipe
+
+      Generate a recipe with example recipe
+      -------------------------
+      #. Source code should be downloadable, as a single file– as an archive (.tar.gz, .zip, .tar.bz2, .tar.xz) or tagged on GitHub,-- toensure that it can be verified. To know why see :ref:`tarballs_no_repos`
+      #. Fork and clone the `staged-recipes <https://github.com/conda-forge/staged-recipes>`_ repository from GitHub.
+      #. Checkout a new branch from the staged-recipes ``main`` branch.
+      #. Through CLI, cd inside the 'staged-recipes/recipes' directory.
+      #. Create a new folder within recipes, for your package (i.e., For unix: ``cp -r recipes/example recipes/<package_name>`` For windows: ``xcopy /h /e /i "recipes\example\" "recipes\<package_name>"``).
+      #. Copy `meta.yaml <https://github.com/conda-forge/staged-recipes/blob/master/recipes/
+        example/meta.yaml>`_ from the example directory. All the changes in the following steps will happen in the COPIED meta.yaml (i.e., `..staged-recipes/recipes/<name-of-package>/meta.yaml`).
+        Please leave the example directory (i.e, ``...staged-recipes/recipes/example/meta.yaml``) unchanged!
+      #. Modify the copied recipe (meta.yaml) as needed. To see how to modify meta.yaml, take a look at :ref:`meta_yaml`.
+      #. Generate the SHA256 key for your source code archive, as described in the example recipe using the ``openssl`` tool. As an alternative, you can also go to the package description on `PyPi <https://pypi.org>`_ from which you can directly copy the SHA256.
```

Comment:
"PyPI"

Also, it would be good to put a comma after the underscore on that PyPI link.